tests/cat/aliases.yml

```diff
@@ -19,5 +19,6 @@ teardown:
 ---
 "cat.aliases":
   - do:
-      cat.aliases: {}
-  - match: { $body.0.alias: 'my_alias' }
+      cat.aliases:
+        format: 'json'
```

Comment:
```suggestion
        format: 'json'
        expand_wildcards: 'open'
```
From the docs: `open` - Match open, non-hidden indices. Also matches any non-hidden data stream. Hopefully this would this fix the issue and allow us to merge this PR.